rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -300,30 +284,32 @@ D3D12_PRIMITIVE_TOPOLOGY get_primitive_topology(u8 draw_mode) noexcept
 	case CELL_GCM_PRIMITIVE_TRIANGLES: return D3D_PRIMITIVE_TOPOLOGY_TRIANGLELIST;
 	case CELL_GCM_PRIMITIVE_TRIANGLE_STRIP: return D3D_PRIMITIVE_TOPOLOGY_TRIANGLESTRIP;
 		// Emulated
-	case CELL_GCM_PRIMITIVE_TRIANGLE_FAN:
+	case CELL_GCM_PRIMITIVE_TRIANGLE_FAN: return D3D_PRIMITIVE_TOPOLOGY_TRIANGLELIST;
```

Comment:
Fix pour How to Survive, c'est le seul triangle qui fait tout affiché correctement, les autre ça fait des bug d'affichage voir totalement écran noir.